### ISSUE TITLE
BUG: init container -n flag required when outputting files format

### ISSUE THREAD

**Issue by DannyeB** [Soluto/kamus#270]
When using the following flags to start the init container I would expect it to mount the secrets as files
```YAML
args: ["-e","/encrypted-secrets","-d","/decrypted-secret-files", "-f", "files"]
```

However, I get the following error:
> Failed to run init container: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undef
ined

Adding the `-n` flag resolves the issue and correctly decrypts the files while ignoring the `-n` 
```YAML
args: ["-e","/encrypted-secrets","-d","/decrypted-secret-files", "-f", "files", "-n", "secrets.env"]
```

**Comment by omerlh** [comment]
Thanks for reporting this! I find the bug and fixing it now

**Comment by omerlh** [comment]
The new init container should be available on dockerhub, using the latest tag or `1.2.0` tag. Please let me know if you're bug wasn't solved.
